packages/nodes-base/nodes/KoboToolbox/descriptions/Operations.ts

```diff
@@ -0,0 +1,130 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const operations = [
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'form',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Get',
+						value: 'get',
+						description: 'Retrieve a form definition',
+					},
+				],
+				default: 'get',
+				description: 'The operation to perform.',
+			},
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'submission',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Query',
+						value: 'query',
+						description: 'Query matching submissions',
+					},
+					{
+						name: 'Get',
+						value: 'get',
+						description: 'Get a single submission',
+					},
+					{
+						name: 'Delete',
+						value: 'delete',
+						description: 'Delete a single submission',
+					},
+					{
+						name: 'Get Validation Status',
+						value: 'get_validation',
+						description: 'Get the validation status for the submission',
+					},
+					{
+						name: 'Update Validation Status',
+						value: 'set_validation',
+						description: 'Set the validation status of the submission',
+					},
+				],
+				default: 'query',
+				description: 'The operation to perform.',
+			},
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'attachment',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Download',
+						value: 'download',
+						description: 'Download all attachments of a given submission',
+					},
+				],
+				default: 'download',
+				description: 'The operation to perform.',
+			},
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'hook',
+						],
+					},
+				},
```

Comment:
Considering that you also did the triggers not sure how useful this operation would be aside from logs and retry

Reply:
Indeed, I'm not too sure either tbh... but it doesn't harm and was easy to implement...